schema.sql

```diff
@@ -33,4 +33,31 @@ alter table animals add column species_id int;
 alter table animals add constraint fk_species foreign key (species_id) references species(id);
 
 alter table animals add column owners_id int;
-alter table animals add constraint fk_owners foreign key ( owners_id) references owners(id);
\ No newline at end of file
+alter table animals add constraint fk_owners foreign key ( owners_id) references owners(id);
+
+create table vet (
```

Comment:
- [ ] Good job so far 👏 But I would like to strongly recommend making sure that your table names be in the plural like in the project requirement, for example, you write **vet** instead of **vets** this can make confusion. Please fix it 👍